plugins/hls-cabal-plugin/test/CabalAdd.hs

```diff
@@ -33,6 +33,8 @@ cabalAddTests =
         (generateAddDependencyTestSession "cabal-add-lib.cabal" ("src" </> "MyLib.hs") "split" [348])
     , runHaskellTestCaseSession "Code Actions - Can add hidden package to a test" ("cabal-add-testdata" </> "cabal-add-tests")
         (generateAddDependencyTestSession "cabal-add-tests.cabal" ("test" </> "Main.hs") "split" [478])
+    , runHaskellTestCaseSession "Code Actions - Can add hidden package to a test with PackageImports" ("cabal-add-testdata" </> "cabal-add-tests")
+        (generateAddDependencyTestSession "cabal-add-tests.cabal" ("test" </> "MainPackageImports.hs") "split" [731])
```

Comment:
Is this single test case enough?  Should I mirror all similar tests?

Reply:
Yeah, sounds reasonable!

Reply:
Sorry, I mean this single test case is enough.